src/core/eme/eme_manager.ts

```diff
@@ -176,13 +186,9 @@ export default function EMEManager(
         wantedSessionType = "persistent-license";
       }
 
-      return observableConcat(serverCertificate$,
-                              getSession(initializationData, stores, wantedSessionType))
+      return getSession(initializationData, stores, wantedSessionType)
         .pipe(mergeMap((sessionEvt) =>  {
           switch (sessionEvt.type) {
-            case "warning":
```

Comment:
So we don't want to return a warning event anymore ?

Reply:
There's none left. The only case was for the server certificate and it was moved to the `mediaKeysInit$` Observable which is already returned in the final merge of the `EMEManager`